miio/chuangmi_ir.py

```diff
@@ -185,6 +204,79 @@ def get_indicator_led(self):
         return self.send("get_indicatorLamp")
 
 
+class ChuangmiRemote(ChuangmiIr):
+    """Class representing new type of Chuangmi IR Remote Controller identified by model
+    "chuangmi-remote-h102a03_".
```

Comment:
```suggestion
    """Class representing Chuangmi IR Remote Controller (chuangmi-remote-h102a03_).
```
Is `info()` working fine for this device? If yes, could you also add that here. The long term goal is to separate the meta information to separate files to allow creating instances based on the model information.

Reply:
I can only confirm that it works for chuangmi-remote-v2, and it does. What should be added? I didn't get. Do you wish to add that `info()` works for the device in the class description?

Reply:
The info will deliver the model of the device, could you check with `miiocli device info` what is it for your device & add that to the corresponding class?

The reason why I'm pushing towards a single class is that it makes it simpler for downstreams like homeassistant to support all supported remotes as long as 1) info is working or 2) the user provides the wanted model as a parameter so that the class can adjust itself accordingly.

Reply:
I get:

```
Model: chuangmi.remote.v2
Hardware version: esp32

Firmware version: 2.0.6_0006
```

The `ChuangmiRemoteV2` child class already states to cover "chuangmi-remote-v2". Do you wish it changed to "chuangmi.remote.v2" according to the model output above? (The default hostname on the network doesn't use dots though, but the actual model name is with dots). I can't confirm anything for the h102a03 model.

Reply:
Ah, yes, it's also in the ttile of the PR, sorry... The one reported by the info is the one to use for identification (as not all devices support the mDNS, which is used for the hostname).